CMakeLists.txt

```diff
@@ -14,9 +14,11 @@ if(NOT CMAKE_OSX_ARCHITECTURES)
     set(CMAKE_OSX_ARCHITECTURES "${CMAKE_HOST_SYSTEM_PROCESSOR}")
 endif()
 
+set(CMAKE_OSX_DEPLOYMENT_TARGET 12)
+
 # Need HOMEBREW_PATH for iso-codes and xkeyboardconfig
 if(CMAKE_OSX_ARCHITECTURES STREQUAL arm64)
-    add_definitions(-target "arm64-apple-macos11")
+    add_definitions(-target "arm64-apple-macos12")
```

Comment:
because of webview

Reply:
Maybe this 12 should be replaced by CMAKE_OSX_DEPLOYMENT_TARGET?